library/posture.c

```diff
@@ -129,7 +129,7 @@ void ziti_posture_init(ziti_context ztx, long interval_secs) {
     }
 
     if (!uv_is_active((uv_handle_t *) ztx->posture_checks->timer)) {
-        uv_timer_start(ztx->posture_checks->timer, ziti_pr_ticker_cb, MILLIS(interval_secs), MILLIS(interval_secs));
+        uv_timer_start(ztx->posture_checks->timer, ziti_pr_ticker_cb, MILLIS(1)/*fire on startup*/, MILLIS(interval_secs));
```

Comment:
You could use `0` here, right?

http://docs.libuv.org/en/v1.x/timer.html#c.uv_timer_start